### ISSUE TITLE
Fbx not displaying correctly with associated errors in VS

### ISSUE THREAD

**Issue by neonzz1** [stride3d/stride#2340]
**Release Type**: Official Release (please choose appropriate option)

**Version**: 4.2.0.2149

**Platform(s)**: Windows
**Describe the bug**
Since upgrading to 4.2.0.2149 my player FBX assets don't display correctly ie floating head gear hair becoming a beard also I occasionally get errors in Visual Studio. 
**To Reproduce**
1. Import the asset and view it in stride.

**Expected behavior**
For the asset to look the same as when viewed in a fbx viewer

**Screenshots**
![image](https://github.com/stride3d/stride/assets/45521010/bd0c4ef8-2ca2-4b3b-a5e6-f1229f230457)
**Log and callstacks**

If any, please attach here any log or callstack (preferably in a `.txt` file using GitHub drag and drop)

**Additional context**
[Wizard.zip](https://github.com/user-attachments/files/15919044/Wizard.zip)

**Comment by Doprez** [comment]
Seems like this is working properly in latest master? 

![image](https://github.com/stride3d/stride/assets/73259914/e72e9637-b395-4810-9835-1fb14b5c4ebf)

![image](https://github.com/stride3d/stride/assets/73259914/6d7d8e0e-c7dd-4514-bcd1-3fe6c239653a)

Import logs from Stride.
```
Error: Texture with name Wizard_Texture not found
Error: Texture with name Wizard_Staff_Texture not found
Info: Computing hashes of 17 source files...
Verbose: Computed hash of C:/dev/stride-dev/testing/MyGame3/MyGame3/Resources/Wizard.fbx for asset Wizard Skeleton. 16 files remaining...
Verbose: Computed hash of C:/dev/stride-dev/testing/MyGame3/MyGame3/Resources/Wizard.fbx for asset Wizard. 15 files remaining...
Verbose: Computed hash of C:/dev/stride-dev/testing/MyGame3/MyGame3/Resources/Wizard.fbx for asset Wizard_CharacterArmature_Death. 14 files remaining...
Verbose: Computed hash of C:/dev/stride-dev/testing/MyGame3/MyGame3/Resources/Wizard.fbx for asset Wizard_CharacterArmature_Idle. 13 files remaining...
Verbose: Computed hash of C:/dev/stride-dev/testing/MyGame3/MyGame3/Resources/Wizard.fbx for asset Wizard_CharacterArmature_Idle_Attacking. 12 files remaining...
Verbose: Computed hash of C:/dev/stride-dev/testing/MyGame3/MyGame3/Resources/Wizard.fbx for asset Wizard_CharacterArmature_Idle_Weapon. 11 files remaining...
Verbose: Computed hash of C:/dev/stride-dev/testing/MyGame3/MyGame3/Resources/Wizard.fbx for asset Wizard_CharacterArmature_PickUp. 10 files remaining...
Verbose: Computed hash of C:/dev/stride-dev/testing/MyGame3/MyGame3/Resources/Wizard.fbx for asset Wizard_CharacterArmature_Punch. 9 files remaining...
Verbose: Computed hash of C:/dev/stride-dev/testing/MyGame3/MyGame3/Resources/Wizard.fbx for asset Wizard_CharacterArmature_RecieveHit. 8 files remaining...
Verbose: Computed hash of C:/dev/stride-dev/testing/MyGame3/MyGame3/Resources/Wizard.fbx for asset Wizard_CharacterArmature_RecieveHit_Attacking. 7 files remaining...
Verbose: Computed hash of C:/dev/stride-dev/testing/MyGame3/MyGame3/Resources/Wizard.fbx for asset Wizard_CharacterArmature_Roll. 6 files remaining...
Verbose: Computed hash of C:/dev/stride-dev/testing/MyGame3/MyGame3/Resources/Wizard.fbx for asset Wizard_CharacterArmature_Run. 5 files remaining...
Verbose: Computed hash of C:/dev/stride-dev/testing/MyGame3/MyGame3/Resources/Wizard.fbx for asset Wizard_CharacterArmature_Run_Weapon. 4 files remaining...
Verbose: Computed hash of C:/dev/stride-dev/testing/MyGame3/MyGame3/Resources/Wizard.fbx for asset Wizard_CharacterArmature_Spell1. 3 files remaining...
Verbose: Computed hash of C:/dev/stride-dev/testing/MyGame3/MyGame3/Resources/Wizard.fbx for asset Wizard_CharacterArmature_Spell2. 2 files remaining...
Verbose: Computed hash of C:/dev/stride-dev/testing/MyGame3/MyGame3/Resources/Wizard.fbx for asset Wizard_CharacterArmature_Staff_Attack. 1 files remaining...
Verbose: Computed hash of C:/dev/stride-dev/testing/MyGame3/MyGame3/Resources/Wizard.fbx for asset Wizard_CharacterArmature_Walk. 0 files remaining...
```


Im not sure if this is normal from before though?
logs from Asset builder when running the game.
```
[(0,0)]: Info: ArgumentOutOfRangeException: Specified argument was out of the range of valid values. (Parameter 'To much bones influencing a single vertex')
[EXEC(0,0)]: Warning: [AssetCompiler] The input file contains vertices that are associated to more than 6 bones. In current version of the system, a single vertex can only be associated to 4 bones. Extra bones will be ignored. Exception: ArgumentOutOfRangeException: Specified argument was out of the range of valid values. (Parameter 'To much bones influencing a single vertex')
```

The mesh still seems fine while static and the idle animation at least still works at runtime soooo not sure what this warning may affect.

**Comment by neonzz1** [comment]
![image](https://github.com/stride3d/stride/assets/45521010/3785a0e8-53c6-47d0-aa29-390615db6268)
This is what I see these assets had already been imported before I updated the engine and after this is what they look like and no before I was getting the texture found. Maybe it's my install of Stride? I have just checked with the master branch of stride and I get this when it's imported 
![image](https://github.com/stride3d/stride/assets/45521010/011bed99-8bf2-462c-a842-09f4721fc703)
And I get this when it is imported.
![image](https://github.com/stride3d/stride/assets/45521010/1a32e686-3e70-4a90-bd97-5700c427858f)

**Comment by Doprez** [comment]
Oh, so there are embedded textures in the FBX file?

Its probably not your install. The FBX importer was completely changed from 4.2.0.2122 to use AssImp instead of the FBX SDK. Master has some fixes for orientation issues like what you were saying with the misplaced hair and beard.

**Comment by Doprez** [comment]
It looks like even in 4.2.0.2122 there is no texture found when I test:
```
Warning: Importer detected a texture not available on disk at address 'C:\Users\trebl\rpg_characters_-_nov_2020\RPG Characters - Nov 2020\Textures\Wizard_Texture.png'
Warning: Importer detected a texture not available on disk at address 'C:\Users\trebl\rpg_characters_-_nov_2020\RPG Characters - Nov 2020\Textures\Wizard_Staff_Texture.png'
```

So Im not sure if its reference something locally for you if its working on your machine.

![image](https://github.com/stride3d/stride/assets/73259914/fa4563aa-4423-43fa-87f5-1b97d4f136c0)

**Comment by neonzz1** [comment]
I clicked "Pack Resources" in blender before I exported it. I just checked and I needed to do a git pull on my end, I had a slightly older version of the master branch. When I open my game with the latest update to the master branch, they display correctly.  And import correctly what can I do to resolve this issue when starting my game, in VS?

**Comment by Doprez** [comment]
Are your packages referencing the source or still 4.2.0.2122? 

In your csproj, make sure the stride versions all reference 4.2.0.1 and do a clean and rebuild. I dont seem to be getting the same issues on my end but it could be one of 2 things I can think of:

- The upgrade itself breaks the Asset file or references somehow.
- its still trying to use 4.2.0.2122 when you run the project.

**Comment by Doprez** [comment]
@neonzz1 can you try the newest release and see if your problems persist? of course make sure to back up your projects but this should have fixed most if not all of the previous problems found with the importer.

**Comment by neonzz1** [comment]
@Doprez  All problems have been solved thank you :)

**Comment by neonzz1** [comment]
Problem was resolved by update to 4.2.0.2188
